hack/lib/common.bash

```diff
@@ -56,3 +56,26 @@ function wait_until_hostname_resolves() {
   logger.error "Timeout waiting for hostname $1 to resolve via DNS"
   return 1
 }
+
+function wait_for_file {
```

Comment:
I suppose we could use something like this (similar to other places where we wait for something)
```
timeout 300 "[[ ! -f "${file} ]]" || return 1
```